test/integration_test/basic_test.go

```diff
@@ -107,6 +107,34 @@ var testStorageClusterBasicCases = []types.TestCase{
 		},
 		TestFunc: BasicUpgradeStorageCluster,
 	},
+	{
+		TestName:        "BasicUpgradeStorageClusterWithAllComponents",
+		TestrailCaseIDs: []string{"C50241"},
+		TestSpec: func(t *testing.T) interface{} {
+			objects, err := ci_utils.ParseSpecs("storagecluster/storagecluster-with-all-components.yaml")
+			require.NoError(t, err)
+			cluster, ok := objects[0].(*corev1.StorageCluster)
+			require.True(t, ok)
+			cluster.Name = "upgrade-stc-with-all-components-test"
+			alertManagerSecret, err := ci_utils.ParseSpecs("monitoring/alertmanager-secret.yaml")
+			require.NoError(t, err)
+
+			logrus.Infof("Creating alert manager secret")
+			err = ci_utils.CreateObjects(alertManagerSecret)
+			require.NoError(t, err)
+			return cluster
+		},
+		ShouldSkip: func(tc *types.TestCase) bool {
+			if len(ci_utils.PxUpgradeHopsURLList) == 0 {
+				logrus.Info("--px-upgrade-hops-url-list is empty, cannot run BasicUpgradeStorageClusterWithAllComponents test")
+				return true
+			}
+			k8sVersion, _ := k8sutil.GetVersion()
+			pxVersion := ci_utils.GetPxVersionFromSpecGenURL(ci_utils.PxUpgradeHopsURLList[0])
+			return k8sVersion.GreaterThanOrEqual(k8sutil.K8sVer1_22) && pxVersion.LessThan(pxVer2_9)
```

Comment:
I don't remember exactly what's the case we want to skip the test. but is this still true?

Reply:
I think PX might have some sort of reqs, I honestly don't remember either, I left it as is for now